File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int,int> freq;
        int freqcnt = 0, sol = INT_MIN;
        for (int i : nums) freq[i]++;
        for (int i : nums) {
            if (freqcnt < freq[i]) {freqcnt = freq[i]; sol = i;}
        }
        return sol;
    }
};

File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(t) != len(s):
            return False;
        freq = {}
        for x in s:
            freq[x] = freq.get(x,0)+1
        for y in t:
            freq[y] = freq.get(y,0)-1
        for x in freq:
            if freq[x] != 0:
                return False
        return True

        